ksql-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -161,6 +165,8 @@ private Object toKsqlValue(final Schema schema,
             schema.valueSchema(), connectSchema.valueSchema(), (Map) convertedValue, pathStr);
       case STRUCT:
         return toKsqlStruct(schema, connectSchema, (Struct) convertedValue, pathStr);
+      case STRING:
+        return String.valueOf(convertedValue);
```

Comment:
Perhaps add a comment in the code that this converts int16, int32, int64, and Boolean keys to string.